ui/src/app/pages/homepage/data-access-requirements.spec.tsx

```diff
@@ -459,4 +465,41 @@ describe('DataAccessRequirements', () => {
         expect(spyCompliance).toHaveBeenCalledTimes(0);
     });
 
+    it ('Should not show Era Commons Module for Registered Tier if the institution does not require eRa', async() => {
```

Comment:
There's an additional requirement that I'd like to see a test for: if the RAS module is not enabled in the environment, then we *always* need to show and require the ERA module for RT.

And now I see that's missing from the AC.  Sorry!  I'll update.

Reply:
I no longer think this is necessary because: the only environment where RAS is not enabled, ERA is also not enabled.

Reply:
So, no change in test correct?

Reply:
right